### PR TITLE
Update README (GA)

### PR DESCRIPTION
**Pull request by misscoded**
### Type of change (place an x in the [ ] that applies)

- [ ] New sample
- [ ] New feature
- [ ] Bug fix
- [x] Documentation

### Summary

Update README with new structure.

### Requirements (place an x in each [ ] that applies)

- [x] I’ve checked my submission against the Samples Checklist to ensure it complies with all standards
- [x] I have ensured the changes I am contributing align with existing patterns and have tested and linted my code
- [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -234,72 +247,75 @@ $ deno test
...
 
-Activity logs for the production instance of your application can be viewed with
-the `slack activity` command:
+Activity logs of your application can be viewed live and as they occur with the
+following command:
 
 ```zsh
-$ slack activity
+$ slack activity --tail
 ```
```
Comment (e-zim) [inline README.md:269]: Great enhancement! 🎉

File: README.md [hunk 2]
```diff
@@ -234,72 +247,75 @@ $ deno test
...
+[Datastores](https://api.slack.com/automation/datastores) securely store data
+for your application on Slack infrastructure. Required scopes to use datastores
+include `datastore:write` and `datastore:read`.
 
-### `/external_auth`
+### `functions/`
 
-[External authentication](https://api.slack.com/future/external-auth) connects
-your app to external services using OAuth2. Once connected, you can perform
-actions as the authorized user on these services from a custom function.
```
Comment (e-zim) [inline README.md:273]: Just checking, but was this section meant to be removed?

File: README.md [hunk 3]
```diff
@@ -99,7 +98,7 @@ client secret and access token.
 > :warning: Running these commands will warn you that a client secret must be
 > added for your OAuth2 provider. We'll take care of this in the next step!
 
-#### Save your Client Secret
+#### Save the Client Secret
```
Comment (e-zim) [inline README.md:101]: Noticed that there's a straggling `(dev)` on L110... Could we include the change to `(local)` in this pull?

File: README.md [hunk 4]
```diff
@@ -1,40 +1,39 @@
-# Deno Timesheet Approval App
+# Timesheet Approval
 
-This app demonstrates a workflow that collects input from a form in Slack and
-saves the results to a preconfigured Google Sheet using external authentication
-in a custom function.
+This automation features a workflow that collects input from a form in Slack and
+saves the results to a Google Sheet.
```
Comment (e-zim) [inline README.md:4]: 📚 👏

**Review by e-zim — APPROVED** [review]
These changes are so refreshing, very nice! 🌟

I left a question towards the end and also noted a small change that I think should be made before merging, but going to give this an approval now, so feel free to merge whenever!

File: README.md (hunk 2 above)
Comment (misscoded) [reply README.md:273]: Absolutely not! Good catch. I entirely forgot this was added. 🤦🏼‍♀️ Are there any other projects that also have this section?

Comment (e-zim) [reply README.md]: No worries! And sorry for a lagging response here! I believe it should also be included on [`deno-github-functions`](https://github.com/slack-samples/deno-github-functions) and [`deno-simple-survey`](https://github.com/slack-samples/deno-simple-survey), even though it's not included on the GitHub functions sample at the moment 😰

**Pull request merged**